app/(main)/home/HomePage.tsx

```diff
@@ -1,44 +1,28 @@
-import CTAButtonSection from "./CTAButtonSection";
 import CarouselContainer from "./CarouselContainer";
-import HomeHeadingCopyText from "./HomeHeadingCopyText";
 import { Button } from "@/components/ui/button";
+import HeadingImage from "./HeadingImage";
+import CTAButtonSection from "./CTAButtonSection";
+import CopyTextParagraph from "./CopyTextParagraph";
+import CopyTextHeader from "./CopyTextHeader";
 
 export default function HomePage() {
   return (
     <>
-      <main className="mb-[150px] flex flex-col items-center justify-center">
-        <HomeHeadingCopyText />
-        <CTAButtonSection />
-        <CarouselContainer />
-      </main>
-      <footer
-        className="flex justify-center px-10
-      sm:mx-0"
-      >
-        <div className="grid grid-flow-row gap-2">
-          <h3 className=" w-[100%] max-w-[565.42px] text-3xl font-bold leading-normal text-[#364071]">
-            Contact Us
-          </h3>
-          <p className="text-sm font-normal leading-normal text-black">
-            Are you a course instructor, student, university admin, or a
-            technologist? We would love to hear from you. Drop us an email at{" "}
-            <Button
-              asChild
-              variant={"link"}
-              className="cursor-pointer p-0 text-blue-500"
-            >
-              <a
-                className="text-sm font-normal leading-normal text-blue-500"
-                href="mailto:here@coursepolicy.ai"
-                target="_blank"
-              >
-                here@coursepolicy.ai
-              </a>
-            </Button>
-          </p>
+      <main className="bg-[#191f3c]">
+        <div className="mx-auto flex max-w-[1600px] justify-between">
+          <div className="ml-[147px]">
+            <CopyTextHeader />
+            <div className="flex md:mt-[20px]">
+              <CopyTextParagraph />
+            </div>
+            <CTAButtonSection />
+          </div>
+          <HeadingImage />
         </div>
-      </footer>
-      <div className="pb-[50px]" />
+      </main>
+      <div className="mx-auto mt-[43px] flex max-w-[1600px] justify-center bg-white">
```

Comment:
things outside of the main div should only be the footer. This carousel needs to be inside the main div. 
```
Header/Navbar
Main
Footer
```